lib/ansible/modules/commands/psexec.py

```diff
@@ -0,0 +1,527 @@
+    connection_username: Administrator
+    connection_password: Password01
+    encrypt: no
+    elevated: yes
+    process_username: Administrator
+    process_password: Password01
+    executable: powershell.exe
+    arguments: (New-Object -ComObject Microsoft.Update.Session).CreateUpdateInstaller().IsBusy
+
+- name: Download and run ConfigureRemotingForAnsible.ps1 to setup WinRM
+  psexec:
+    hostname: windows-pc
+    connection_username: Administrator
+    connection_password: Password01
+    encrypt: yes
+    executable: powershell.exe
+    arguments: '-'
+    stdin: |
+      $ErrorActionPreference = "Stop"
+      $sec_protocols = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::SystemDefault
+      $sec_protocols = $sec_protocols -bor [Net.SecurityProtocolType]::Tls12
+      [Net.ServicePointManager]::SecurityProtocol = $sec_protocols
+      $url = "https://github.com/ansible/ansible/raw/devel/examples/scripts/ConfigureRemotingForAnsible.ps1"
+      Invoke-Expression ((New-Object Net.WebClient).DownloadString($url))
+      exit
+'''
+
+RETURN = r'''
+msg:
+  description: Any exception details when trying to run the process
+  returned: module failed
+  type: str
+  sample: 'Received exception from remote PAExec service: Failed to start "invalid.exe". The system cannot find the file specified. [Err=0x2, 2]'
+stdout:
+  description: The stdout from the remote process
+  returned: success and interactive or asynchronous is 'no'
+  type: str
+  sample: Hello World
+stderr:
+  description: The stderr from the remote process
+  returned: success and interactive or asynchronous is 'no'
+  type: str
+  sample: Error [10] running process
+pid:
+  description: The process ID of the asynchronous process that was created
+  returned: success and asynchronous is 'yes'
+  type: int
+  sample: 719
+rc:
+  description: The return code of the remote process
+  returned: success and asynchronous is 'no'
+  type: int
+  sample: 0
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_bytes, to_native, to_text
+
+try:
+    from pypsexec import client
+    from pypsexec.exceptions import PypsexecException, PAExecException, \
+        PDUException, SCMRException
+    from pypsexec.paexec import ProcessPriority
+    from smbprotocol.exceptions import SMBException, SMBAuthenticationError, \
+        SMBResponseException
+    HAS_PYPSEXEC = True
+except ImportError:
+    HAS_PYPSEXEC = False
+
+try:
+    import gssapi
+    # GSSAPI extension required for Kerberos Auth in SMB
+    from gssapi.raw import inquire_sec_context_by_oid
+    HAS_KERBEROS = True
+except ImportError:
+    HAS_KERBEROS = False
+
+
+def remove_artifacts(module, client):
+    warning_msg = "Failed to cleanup PAExec service and executable"
+    try:
+        client.remove_service()
+    except PDUException as exc:
+        module.warn("%s with RPC PDU fault: %s" % (warning_msg, to_text(exc)))
```

Comment:
Any reason you're not using `format()`? This works, but is kind of "old style" for new code.